properties.yaml

```diff
@@ -175,6 +175,15 @@ DevToolsDetached:
   documentation: |
     If enabled, dev tools window will be detached from the main application
 
+DevToolsAnimationsEnabled:
+  key: compose.reload.devToolsAnimationsEnabled
```

Comment:
What are the cases where people would just want to use this property for their project?

Reply:
I think mainly this property can be useful for cases when transparency is disabled (e.g. like in #300) If people does not want to wait for animations and wait for the white background --- they can choose to turn off the animations